pycqed/measurement/calibration/single_qubit_gates.py

```diff
@@ -368,7 +370,7 @@ def run_analysis(self, **kw):
         """
         self.analysis = tda.FluxPulseScopeAnalysis(
             qb_names=self.meas_obj_names,
-            options_dict=dict(TwoD=True, global_PCA=True,))
+            options_dict=dict(TwoD=True, rotation_type='global_PCA'))
```

Comment:
should be 'fixed_cal_points' is cal states are used (and global_PCA otherwise). or maybe allow the user to pass a choice via analysis_kwargs (defaulting to what I have suggested in the previous sentence)?

Reply:
Fixed now